lib/chef/resource/windows_audit_policy.rb

```diff
@@ -225,6 +201,32 @@ def option_configured?(option_name, option_setting)
           end
         end
       end
+
+      action_class do
+        def subcategory_configured?(sub_cat, success_value, failure_value)
+          setting = if success_value && failure_value
+                      "Success and Failure$"
+                    elsif success_value && !failure_value
+                      "Success$"
+                    elsif !success_value && failure_value
+                      "#{sub_cat} \\s+ Failure$"
```

Comment:
```suggestion
                      "#{sub_cat}\\s+Failure$"
```

This was saying there should always be 3+ spaces in the middle. Was that intentional?